src/Microsoft.Crank.Jobs.PipeliningClient/pipelining.yml

```diff
@@ -26,4 +26,5 @@ jobs:
       serverAddress: localhost
       serverPort: 5000
       path: /
-    arguments: '--url "{{serverScheme}}://{{serverAddress}}:{{serverPort}}{{path}}" --connections {{connections}} --warmup {{warmup}} --duration {{duration}} --pipeline {{pipeline}} {{headers[presetHeaders]}}'
+      customHeaders: [ ] # list of headers with the format: '<name1>: <value1>', e.g. [ 'content-type: application/json' ]. In yml should look like: - "content-type: application/json"
+    arguments: "--url \"{{serverScheme}}://{{serverAddress}}:{{serverPort}}{{path}}\" --connections {{connections}} --warmup {{warmup}} --duration {{duration}} --pipeline {{pipeline}} {{headers[presetHeaders]}} {% for h in customHeaders %}{% assign s = h | split : ':' %}--header \"{{ s[0] }}: {{ s[1] | strip }}\" {% endfor %}"
```

Comment:
Did you check the arguments with the app first, that it does what you want when you pass the values like this on the command line?

Reply:
i checked - it using `--url "https://127.0.0.1:5000/hello-world" --connections 1 --duration 15 --header "Invalid Header: value" --detailedResponseStats` run, so it works now. added PR description with what I have changed